nautobot_ssot/templates/nautobot_ssot_servicenow/config.html

```diff
@@ -0,0 +1,12 @@
+{% extends "generic/object_edit.html" %}
+
+{% block header %}
+    {% if settings.PLUGINS_CONFIG.nautobot_ssot.servicenow_instance or settings.PLUGINS_CONFIG.nautobot_ssot.servicenow_username or settings.PLUGINS_CONFIG.nautobot_ssot.servicenow_password %}
```

Comment:
This might be better implemented via a call to the `messages` API in the view code, rather than hard-coded logic in the page template? Non-blocking.